source/github-events/on-diff-file-load.ts

```diff
@@ -4,13 +4,11 @@ import delegate from 'delegate-it';
 const fragmentSelector = [
 	'include-fragment.diff-progressive-loader', // Incremental file loader on scroll
 	'include-fragment.js-diff-entry-loader', // File diff loader on clicking "Load Diff"
-].join();
+].join(',');
 
 // This lets you call `onDiffFileLoad` multiple times with the same callback but only ever a `load` listener is registered
-const getDeduplicatedHandler = mem((callback: EventListener): delegate.EventHandler => {
-	return (event: delegate.Event) => {
-		event.delegateTarget.addEventListener('load', callback);
-	};
+const getDeduplicatedHandler = mem((callback: EventListener): delegate.EventHandler => (event: delegate.Event) => {
```

Comment:
This is a little hard to see on one line. Should it at least be this?
```suggestion
const getDeduplicatedHandler = mem((callback: EventListener): delegate.EventHandler =>
	(event: delegate.Event) => {
```

Or even what Prettier probably suggests:
```suggestion
const getDeduplicatedHandler = mem(
	(callback: EventListener): delegate.EventHandler =>
		(event: delegate.Event) => {
```